src/Home.js

```diff
@@ -2,16 +2,27 @@ import React from 'react';
 import ModulePlan from './ModulePlan';
 import PlanningSection from './PlanningSection';
 import moduleplan from './moduleplan.json';
+import users from './users.json';
 
 class Home extends React.Component {
   render() {
```

Comment:
I think it's okay when it stays where it is right now because `getSemestersForUser` is specific for `Home`. It makes sense to move it to another file/module if that method is needed in another module or class as well.